src/main/common/filter.c

```diff
@@ -54,6 +54,7 @@ void pt1FilterInitRC(pt1Filter_t *filter, float tau, float dT)
     filter->state = 0.0f;
     filter->RC = tau;
     filter->dT = dT;
+    filter->k = filter->dT / (filter->RC + filter->dT);
```

Comment:
That parameter is commonly called `alpha` IIRC. Since we're using conventional `RC` and `dT`, we should probably stick to convention in naming other values here. WDYT?

Reply:
good idea, let me change that